photutils/psf/tests/test_groupstars.py

```diff
@@ -0,0 +1,346 @@
+            |                             *                            |
+          0 +                             *                            +
+            +--------------+--------------+-------------+--------------+
+           -1            -0.5             0            0.5             1
+        """
+
+        first_group = Table([np.zeros(5), np.linspace(0, 1, 5),
+                             np.arange(5) + 1, np.ones(5, dtype=np.int)],
+                            names=('x_0', 'y_0', 'id', 'group_id'))
+        second_group = Table([np.zeros(5), np.linspace(2, 3, 5),
+                              6 + np.arange(5), 2*np.ones(5, dtype=np.int)],
+                             names=('x_0', 'y_0', 'id', 'group_id'))
+        starlist = vstack([first_group, second_group])
+        daogroup = DAOGroup(crit_separation=0.3)
+        test_starlist = daogroup(starlist['x_0', 'y_0', 'id'])
+        assert_array_equal(starlist, test_starlist)
+
+    def test_daogroup_three(self):
+        """
+           1 +--+-------+--------+--------+--------+-------+--------+--+
+             |                                                         |
+             |                                                         |
+             |                                                         |
+         0.5 +                                                         +
+             |                                                         |
+             |                                                         |
+           0 +  *   *   *    *   *                 *   *   *    *   *  +
+             |                                                         |
+             |                                                         |
+        -0.5 +                                                         +
+             |                                                         |
+             |                                                         |
+             |                                                         |
+          -1 +--+-------+--------+--------+--------+-------+--------+--+
+                0      0.5       1       1.5       2      2.5       3
+        """
+
+        first_group = Table([np.linspace(0, 1, 5), np.zeros(5),
+                             np.arange(5) + 1, np.ones(5, dtype=np.int)],
+                            names=('x_0', 'y_0', 'id', 'group_id'))
+        second_group = Table([np.linspace(2, 3, 5), np.zeros(5),
+                              6 + np.arange(5), 2*np.ones(5, dtype=np.int)],
+                             names=('x_0', 'y_0', 'id', 'group_id'))
+        starlist = vstack([first_group, second_group])
+        daogroup = DAOGroup(crit_separation=0.3)
+        test_starlist = daogroup(starlist['x_0', 'y_0', 'id'])
+        assert_array_equal(starlist, test_starlist)
+
+    def test_daogroup_four(self):
+        """
+             +-+---------+---------+---------+---------+-+
```

Comment:
I said this in the telecon... But I *love* these diagrams.  No changes needed, just positive feedback ;)